content/english/blog/road_to_ubiquity.md

```diff
@@ -0,0 +1,75 @@
+---
+title : "WebAssembly and the Road to Ubiquity"
+image : "images/blogs/ij_map_sample.jpeg"
+date: 2022-09-19T9:00:00-04:00
+author: "Kevin Hoffman"
+author_profile: "https://www.linkedin.com/in/%F0%9F%A6%80-kevin-hoffman-9252669/"
+description : "A brief look down WebAssembly's road to ubiquity"
+categories: ["webassembly", "wasmcloud", "ubiquity", "wasi", "component model"]
+draft : false
+---
+
+It may seem odd or counter-intuitive, but most of us within the WebAssembly community are eagerly awaiting the day that WebAssembly becomes _"boring"_. Choosing so-called boring technology is a good, safe bet for building production systems. Boring technology does what it's supposed to, it's easy to work with, it doesn't crash or break down, and has a simple developer experience. This is what we want WebAssembly to be: boring and **_ubiquitous_**.
+
+Unfortunately, we're not there yet. WebAssembly is far from boring these days. Building a `.wasm`  module involves a lot of bespoke tooling, knowledge (much of which is tribal or hard to discover), plugins, and extensions. The experience varies between languages, and, much to our chagrin, even operating systems. Worse, the experience also varies whether you're targeting a browser or not, despite our protests that the browser should never be treated as a "more equal" citizen in the wasm community. Working in the WebAssembly ecosystem today involves a number of high-caliber footguns.
+
+In the not-so-distant future, WebAssembly will cease to become an end goal and will instead become a simple implementation detail; a mere checkbox or command-line flag. Developers using Apple's Xcode tools will create a project and check a box indicating that they're targeting the build as a WebAssembly Component. VScode and other IDEs will have integrated support for components that expose their interfaces through the component model's `.wit`  (or whatever it becomes next) definitions, so people will get full syntax highlighting and type checking when they create a component that relies upon another component. There will be dependency visualization tools showing the chain of required components. We will have public repositories, artifactory, and other repository vendors will all natively support storing, querying, and annotating WebAssembly modules.
```

Comment:
There's a significant tone shift here, from gloomy present-day in the previous paragraph to pleasantly boring future. Consider adding a transition word/phrase/sentence to smooth that shift